### PR TITLE
Contribution docs

### PR DESCRIPTION
**Pull request by mokeefe09**
I am a student that was tasked with making a pull request to a real open-source project for a class. I made some changes to the documentation on how to contribute to the project. I tried to make the contribution instructions more organized and easier to understand for new/beginner contributors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+    - [Reporting Bugs](#42-reporting-bugs)
+    - [Fixing Bugs](#43-fixing-bugs)
+    - [Translation](#44-translation)
+    - [Graphics](#45-graphics)
+    - [Audio](#46-audio)
+    - [Scenarios](#47-scenarios)
+- [Best Practices](#5-best-practices)
+    - [Adding New Strings](#51-adding-new-strings)
+    - [Coding Style](#52-coding-style)
+    - [Language](#53)
```
Comment (tupaschoal) [inline CONTRIBUTING.md:17]: This link is broken, it probably should have been:
```suggestion
    - [Language](#53-language)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,12 +1,39 @@
...
+
+# 2. Contribution Philosophy
+Any contribution to OpenRCT2 is welcome and valued. Contributions can be in the form of bug reports, translation or code additions / changes. Please read this document and the [How To Contribute](https://github.com/OpenRCT2/OpenRCT2/wiki/How-To-Contribute) section of the wiki to learn how to contribute as effectively as possible. If you have any questions or concerns, please ask in the [Discord chat](https://discordapp.com/invite/fsEwSWs).
+
+# 3. Workflow
+OpenRCT2 uses the [gitflow workflow](https://www.atlassian.com/git/tutorials/comparing-workflows#gitflow-workflow). If you are implementing a new feature or logic from the original game, please branch off and perform pull requests to ```develop```. If you are fixing a bug for the next release, please branch off and perform pull requests to the correct release branch. ```master``` only contains tagged releases, you should never branch off this.
 questions or concerns, please ask in the [Discord chat](https://discordapp.com/invite/fsEwSWs).
```
Comment (tupaschoal) [inline CONTRIBUTING.md:28]: Seems like this line 28 was a leftover from the item #2

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -18,19 +45,27 @@ by running:
...
 ```
 
-For Windows builds, OpenRCT2 will generate a memory dump and saved game when the game crashes unexpectedly. The game
-will open explorer to these files automatically for you. They are placed inside your configured user directory which
-by default is `%HOMEPATH%\Documents\OpenRCT2`.
+For Windows builds, OpenRCT2 will generate a memory dump and saved game when the game crashes unexpectedly. The game will open explorer to these files automatically for you. They are placed inside your configured user directory which by default is `%HOMEPATH%\Documents\OpenRCT2`.
+
+
+## 4.3 Fixing bugs
+A list of bugs can be found on the [issue tracker](https://github.com/OpenRCT2/OpenRCT2/issues). Feel free to work on any bug and submit a pull request to the develop branch with the fix. Mentioning that you intend to fix a bug on the issue will prevent other people from trying as well.
```
Comment (tupaschoal) [inline CONTRIBUTING.md:52]: Can you elaborate on having moved bug fixes to the third in the list?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -18,19 +45,27 @@ by running:
...
+
+## 4.4 Translation
+You can translate the game into other languages by editing the language files in ```data/language``` directory. Please join discussions in the [#localisation channel on Discord](https://discordapp.com/invite/sxnrvX9) and submit pull requests to [OpenRCT2/Localisation](https://github.com/OpenRCT2/Localisation).
+
+## 4.5 Graphics
+You can help create new graphics for the game by visiting the [OpenGraphics project](https://github.com/OpenRCT2/OpenGraphics). 3D modellers needed!
+
+## 4.6 Audio
+You can help create the music and sound effects for the game. Check out the OpenMusic repository and drop by our [#open-sound-and-music channel on Discord](https://discord.gg/9y8WbcX) to find out more.
```
Comment (tupaschoal) [inline CONTRIBUTING.md:61]: While you're at it, I think the `OpenMusic` repository could use a hyperlink.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -18,19 +45,27 @@ by running:
...
 
-# Translation
-Translation is managed in a separate repository, [OpenRCT2/Localisation](https://github.com/OpenRCT2/Localisation).
-You will find more information there.
+## 4.7 Scenarios
+We would also like to distribute additional scenarios with the game, when the time comes. For that, we need talented scenario makers! Check out the [OpenScenarios repository](https://github.com/PFCKrutonium/OpenRCT2-OpenScenarios).
```
Comment (tupaschoal) [inline CONTRIBUTING.md:64]: This repository has been moved into the organization, so  the link can be updated
```suggestion
We would also like to distribute additional scenarios with the game, when the time comes. For that, we need talented scenario makers! Check out the [OpenScenarios repository](https://github.com/OpenRCT2/OpenScenarios).
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -41,9 +76,9 @@ important that you create an issue as this can be more easily overlooked.
...
 When coding, please also add a string constant for your strings to ```./src/localisation/StringIds.h```.
 
-### Coding style
+## 5.2 Coding style
 Use [this](https://github.com/OpenRCT2/OpenRCT2/wiki/Coding-Style) code style as a reference for new or changed code.
 
-### Language
+## 5.3 Language
 For now, it is recommended that you only write C++ files as the majority of the game is currently in
 C++. Exceptions are to modules that have direct relationship to original code.
```
Comment (tupaschoal) [inline CONTRIBUTING.md:84]: I think this whole section could be dropped, or rewritten to say it's C++ only except when dealing with the plugins files?

What do you think @OpenRCT2/developers ?

File: readme.md [hunk 7]
```diff
@@ -212,28 +206,11 @@ cmake --build . --target install
...
-
-## 4.5 Audio
-You can help create the music and sound effects for the game. Check out the OpenMusic repository and drop by our [#open-sound-and-music channel on Discord](https://discord.gg/9y8WbcX) to find out more.
-
-## 4.6 Scenarios
-We would also like to distribute additional scenarios with the game, when the time comes. For that, we need talented scenario makers! Check out the [OpenScenarios repository](https://github.com/PFCKrutonium/OpenRCT2-OpenScenarios).
+There are two places to find information on how to contribute:
+- See [CONTRIBUTING.md](https://github.com/OpenRCT2/OpenRCT2/blob/develop/CONTRIBUTING.md) for a brief, high-level overview of the different types of contributions relevant to the project. There are also recommendations regarding coding style and best practices.
 
+- See the [How To Contribute](https://github.com/OpenRCT2/OpenRCT2/wiki/How-To-Contribute) section of the wiki for a more in-depth explanation of the concrete steps required to contribute (forking, cloning, pull requests, etc.). The wiki contains other documentation contributors may find useful, such a detailed description of data structures used in the game.
```
Comment (tupaschoal) [inline readme.md:213]:
```suggestion
- See the [How To Contribute](https://github.com/OpenRCT2/OpenRCT2/wiki/How-To-Contribute) section of the wiki for a more in-depth explanation of the concrete steps required to contribute (forking, cloning, pull requests, etc.). The wiki contains other documentation contributors may find useful, such as detailed description of data structures used in the game.
```

Comment (Gymnasiast) [reply readme.md:213]:
```suggestion
- See the [How To Contribute](https://github.com/OpenRCT2/OpenRCT2/wiki/How-To-Contribute) section of the wiki for a more in-depth explanation of the concrete steps required to contribute (forking, cloning, pull requests, etc.). The wiki contains other documentation contributors may find useful, such as detailed descriptions of data structures used in the game.
```

**Comment by tupaschoal** [comment]
@mokeefe09 are you intending to finish this task?

**Comment by tupaschoal** [comment]
@Gymnasiast I've tended to our comments so we can merge this, can you re-review, please?

**Comment by Gymnasiast** [comment]
@tupaschoal I think the most important thing to consider here is whether we want all this information to be in the readme, or to move most of it to the wiki (and point to there from the readme). I’m personally in favour of the latter, as it makes it much easier to update this information.

**Review by tupaschoal — COMMENTED** [review]
I agree, though I think this PR has value already, since it reduces the size of the main README and concentrates contributing related stuff on the corresponding contributing file

**Pull request closed without merging**